coders/heic.c

```diff
@@ -862,86 +999,55 @@ static MagickBooleanType WriteHEICImage(const ImageInfo *image_info,
     status=IsHeifSuccess(&error,image,exception);
     if (status == MagickFalse)
       break;
-    /*
+
+    printf("Colorspace: %d\n", image->colorspace);
+    if (image->colorspace == sRGBColorspace) {
```

Comment:
You could also use `IssRGBCompatibleColorspace(image->colorspace) != MagickFalse` here instead.

Reply:
And another thing I just though of is that we should only write an RGB/RGBA image when the output format is AVIF.